.go-version

```diff
@@ -1,2 +1,2 @@
-1.21.13
+1.22
```

Comment:
The `.go-version` file should be a full version name for the toolchain to use for building the plugin at release-time.
Since we require 1.22 with the last changes to the `go.mod`, I suggest we settle on the latest release on this branch, so `1.22.8` here.

Reply:
Resolved in https://github.com/hashicorp/packer-plugin-vsphere/commit/80f6f37135819f0211f210761f46108b7d264faa.